nil pointer reference when no song is playing

Attempting to do a nil check on a parameter of `currentlyPlaying` causes nil reference when `currentlyPlaying` is nil

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0x66acd2]

goroutine 1 [running]:
main.main()
Slackify/main.go:36 +0x152
exit status 2
```

--- a/main.go
+++ b/main.go
@@ -33,23 +33,26 @@ func main() {
 		// Get currentlyPlaying data from spotify:
 		currentlyPlaying := GetCurrentlyPlaying(client)
 
-		if currentlyPlaying.Item != nil {
-			// Create a formatted status update
-			status := fmt.Sprintf("Currently Listening to: %s - %s", currentlyPlaying.Item.Artists[0].Name, currentlyPlaying.Item.Name)
+		if currentlyPlaying != nil {
+			if currentlyPlaying.Item != nil {
+				// Create a formatted status update
+				status := fmt.Sprintf("Currently Listening to: %s - %s", currentlyPlaying.Item.Artists[0].Name, currentlyPlaying.Item.Name)
 
-			// Get the progress (current timestamp) of the track we're listening to:
-			progress := currentlyPlaying.Progress
-			// Get the full length of the track we're listening to:
-			length := currentlyPlaying.Item.Duration
+				// Get the progress (current timestamp) of the track we're listening to:
+				progress := currentlyPlaying.Progress
+				// Get the full length of the track we're listening to:
+				length := currentlyPlaying.Item.Duration
 
-			// Get the time remaining of the track in miliseconds
-			delay = length - progress + 3
+				// Get the time remaining of the track in miliseconds
+				delay = length - progress + 3
 
-			// Output to terminal our current status, here \r ensures only use a single line
-			fmt.Printf("\r " + status)
+				// Output to terminal our current status, here \r ensures only use a single line
+				fmt.Printf("\r " + status)
 
-			// Send our status to slack:
-			setSlackStatus(status, delay)
+				// Send our status to slack:
+				setSlackStatus(status, delay)
+			}
+
 		} else {
 			// We're not listening to anything, Tell the user;
 			fmt.Printf("\r Nothing playing...")

